README.md

```diff
@@ -33,60 +33,59 @@ docker run --privileged --rm linuxkit/binfmt:v0.8
 ```
 sudo snap install yq
 ```
-5. Install essentials build packages
+5. Install deno
+```
+sudo snap install deno
+```
+6. Install essentials build packages
 ```
 sudo apt-get install -y build-essential openssl libssl-dev libc6-dev clang libclang-dev ca-certificates
 ```
-6. Install Rust
+7. Install Rust
 ```
 curl https://sh.rustup.rs -sSf | sh
 # Choose nr 1 (default install)
 source $HOME/.cargo/env
 ```
-7. Install toml
-```
-cargo install toml-cli
-```
 8. Build and install embassy-sdk
 ```
-cd ~/ && git clone https://github.com/Start9Labs/embassy-os.git
+cd ~/ && git clone --recursive https://github.com/Start9Labs/embassy-os.git
 cd embassy-os/backend/
 ./install-sdk.sh
+embassy-sdk init
 ```
+Now you are ready to build your **hello-world** service
 
 ## Cloning
 
-Clone the project locally. Note the submodule link to the original project(s). 
+Clone the project locally. 
 
 ```
 git clone https://github.com/Start9Labs/hello-world-wrapper.git
 cd hello-world-wrapper
 ```
-During the 0.3.0 beta (you won't need this after merge to main), run:  
-```
-git checkout integration/0.3.0
-```
+
 ## Building
 
-To build the project, run the following commands:
+To build the **Hello World** service, run the following command:
 
 ```
 make
 ```
 
 ## Installing (on Embassy)
 
-SSH into an Embassy device.
-`scp` the `.s9pk` to any directory from your local machine.
-Run the following command to install the package:
+Run the following commands to determine successful install:
+> :information_source: Change embassy-q1w2e3r4.local to your Embassy address
 
 ```
 embassy-cli auth login
-#Enter your embassy password then run:
-embassy-cli package install /path/to/hello-world.s9pk
+#Enter your embassy password
+embassy-cli --host https://embassy-q1w2e3r4.local package install hello-world.s9pk
```

Comment:
you prob don't want to use an actual .local for the hello-world public repo README

Reply:
oh I just realized its qwerty with numbers lol - maybe explain in the comments that its just an example name?

Reply:
Hey Dread thanks for reviewing 🙏
You have missed the line [79](https://github.com/Start9Labs/hello-world-wrapper/blob/8e45a7deafcdf057daf55ab7b982a55798a63f6c/README.md?plain=1#L79):
 > :information_source: Change embassy-q1w2e3r4.local to your Embassy address

Reply:
Awesome, Thanks!